pkg/chunk/gcp/instrumentation.go

```diff
@@ -0,0 +1,49 @@
+package gcp
+
+import (
+	"time"
+
+	"github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc"
+	"github.com/mwitkow/go-grpc-middleware"
+	"github.com/opentracing/opentracing-go"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/weaveworks/common/instrument"
+	"golang.org/x/net/context"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
+)
+
+var bigtableRequestDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	Namespace: "cortex",
+	Name:      "bigtable_request_duration_seconds",
+	Help:      "Time spent doing BigTable requests.",
+
+	// DynamoDB latency seems to range from a few ms to a few sec and is
+	// important.  So use 8 buckets from 64us to 8s.
+	Buckets: prometheus.ExponentialBuckets(0.000128, 4, 8),
```

Comment:
Either comment or call to `ExponentialBuckets` is wrong. When I evaluate, I get:

```
[0.000128, 0.000512, 0.002048, 0.008192, 0.032768, 0.131072, 0.524288, 2.097152]
```

So:
- not 8s
- not from 64us